statsmodels/tsa/tests/test_tsa_indexes.py

```diff
@@ -17,6 +17,16 @@
 import pandas as pd
 import pytest
 
+# RangeIndex only introduced in Pandas 0.18, so we include a shim until
+# Statsmodels requires that version.
```

Comment:
what version _does_ statsmodels require?  setup.py says 0.13, INSTALL.txt says 0.14, and both of those are absurdly old (0.23.0 is being released before long)

Reply:
I think I remember that the policy is that we support whatever is in Travis, so that would be 0.14. I think after 0.9 the plan is to boost the minimums. For example, we will at least have a minimum of scipy = 0.16 so that we can drop the compatibility layer for state space models. I don't know if there's a particular version target for Pandas.